Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -10,7 +10,7 @@ class ChannelDTO: NSManagedObject {
     @NSManaged var cid: String
     @NSManaged var name: String?
     @NSManaged var imageURL: URL?
-    @NSManaged var typeRawValue: String
+    @NSManaged var typeRawValue: String?
```

Comment:
maybe better if we do the way around? (saving a channel without its type does not make a lot of sense)